content/show/coder-radio/0001.md

```diff
@@ -0,0 +1,71 @@
+{
+  "type": "episode",
+  "draft": false,
+  "show_slug": "coder-radio",
+  "show_name": "Coder Radio",
+  "episode": 1,
+  "episode_padded": "0001",
+  "episode_guid": "00000000-0000-0000-0000-000000000000",
+  "slug": "1",
+  "title": "Gateways to Programming",
+  "description": "Michael and Chris introduce our new weekly software development podcast. We start with a look at ways beginners can get started with development. Plus we chat about the issues new developers face entering a market dominated by App stores. Then \u2013  How platform vendors are feeling the need to reclaim greater control from developers.",
+  "date": "2012-06-11",
+  "header_image": "/images/shows/coder-radio.png",
+  "categories": [
+    "Coder Radio"
+  ],
+  "tags": [],
+  "hosts": [
+    "chris",
+    "michael"
+  ],
+  "guests": [],
+  "sponsors": [],
+  "podcast_duration": null,
+  "podcast_file": null,
+  "podcast_bytes": null,
+  "podcast_chapters": null,
+  "podcast_alt_file": "https://traffic.libsyn.com/jnite/cr-001-mp3.mp3",
+  "podcast_ogg_file": "https://traffic.libsyn.com/jnite/cr-001-ogg.ogg",
+  "video_file": "https://201206.jb-dl.cdn.scaleengine.net/coderradio/2012/cr-001-432p.mp4",
+  "video_hd_file": null,
+  "video_mobile_file": null,
+  "youtube_link": "https://youtu.be/02OnmKa4vg4",
+  "jb_url": "/20497/gateways-to-programming-cr-01/index.html",
+  "fireside_url": "/1"
+}
+
+
+### Episode Links
+
+## Show Notes:
+
+#### What is Coder Radio?
```

Comment:
For Episodes that have both `Show Notes` and `Episode Links`, one should be removed so we don't have multiple titles for the same content.